site/styles/_availability-tags.scss

```diff
@@ -1,43 +1,63 @@
-s72-availability-status,
 s72-availability-label {
-  position: absolute;
-  z-index: 99;
-}
-
-.availability-bg {
-  background: rgba(0, 0, 0, 0.8);
-  display: block;
-  padding: 5px 8px;
-  text-align: center;
-
+  font-weight: $font-weight-light;
   .availability-state {
-    font-size: 12px;
-    font-weight: $font-weight-bold;
-    text-transform: none;
-  }
-}
+    @extend .d-flex;
+    line-height: 24px;
+    @extend .mb-2;
+    font-size: 16px;
+    letter-spacing: 0.4px;
 
-.availability-bg,
-.availability-state {
-  font-size: 12px;
-}
-
-.availability-state {
-  color: #fff;
-  font-weight: $font-weight-bold;
-  text-transform: uppercase;
+    &:before {
+      @extend .fa;
+      @extend .fa-exclamation-circle;
+      @extend .mr-1;
+      color: var(--body-color);
+      line-height: 24px;
+      font-size: 24px;
+    }
+  }
 }
 
 .poster {
+  s72-availability-status,
   s72-availability-label {
-    left: 0 !important;
-    margin-left: auto;
-    margin-right: auto;
-    padding: 10px 10px 5px;
-    right: 0;
+    position: absolute;
+    z-index: 99;
+  }
+
+  s72-availability-label {
+    background-color: rgba(var(--body-bg-rgb),0.7);
+    font-weight: $font-weight-normal;
+    @extend .w-100;
+    @extend .px-p5;
+    bottom: 0;
 
     .availability-state {
-      text-align: center;
+      line-height: 18px;
+      @extend .mb-0;
+      &:before {
+        content: none;
+      }
     }
   }
+
+  s72-availability-status .availability-bg {
```

Comment:
![image](https://user-images.githubusercontent.com/10665859/160332348-2c2f05cd-dcc9-4eee-b935-653750507b1b.png)

This is different to the designs:
![image](https://user-images.githubusercontent.com/10665859/160332377-b04e7ef5-4a62-4555-bbfd-e3ab3a7fdf76.png)

It's also broken on the detail page:
![image](https://user-images.githubusercontent.com/10665859/160332444-237e4e6b-eed6-4367-aaa5-9cc1b94d0309.png)

Reply:
I'm trying to feed the overall changes in incrementally, but I think you're right, it should look like the design as I've touched it in this PR, therefore:

- Have matched it to the design now
- This PR now also updates the sitewide button bg/text color (and this label now inherits those colors)
-  That broken detail page appears in develop branch, do you reckon we should fix it though? As the label in screenshot is artificially long. If so is it cool if I fix it in an upcoming future branch (It updates the detail page poster widths to the designs)?

(Context: This is [the branch](https://github.com/shift72/core-template/tree/Implement-the-rest-of-the-designs-6846) discussed in our interteam collab with all changes (Feeding in as smaller PRs))

Reply:
IMO it's worth fixing now... clients can put whatever they want in those translations so I think we should be handling extra long labels.

Reply:
Oh yes good point. Fix now applied.

Reply:
Much better!
![image](https://user-images.githubusercontent.com/10665859/160759705-3906bf74-5649-4a0b-af75-34eeb33c759b.png)